cmd/atlas/internal/cmdapi/flag.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package cmdapi
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	flagAutoApprove = "auto-approve"
+	flagDevURL      = "dev-url"
+	flagDryRun      = "dry-run"
+	flagDSN         = "dsn" // deprecated in favor of flagURL
+	flagExclude     = "exclude"
+	flagFile        = "file"
+	flagSchema      = "schema"
+	flagURL         = "url"
+	flagVar         = "var"
+)
```

Comment:
I don't see a need for this file. Some flags can reside in project.go and others in schema.go/cmdapi.go.